production-vms/resources.go

```diff
@@ -0,0 +1,175 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"production-vms/config"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/to"
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/compute/armcompute"
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/network/armnetwork"
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/resources/armresources"
+)
+
+func createProductionVM(ctx context.Context, vhd *VHD, subnetID string) error {
```

Comment:
are we planning on running this in production subscriptions?

Reply:
Yep the goal will be for these VMs to be running the images we release

Reply:
well the idea would be that we could run those VMs in any subscriptions right ? does that require us to have the subID in AME ?

Reply:
Yes it should be very flexible to any subscription. I think it will mean that we will need a subID and rg in AME - unless we can get away with using msft? (since its the same images, just the vms would be running somewhere else. But for cost tracking / optics of saying "these are the prod images in prod" might mean we want it in AME)